src/queries/placeholder-text.js

```diff
@@ -1,7 +1,8 @@
 import {queryAllByAttribute, buildQueries} from './all-utils'
 
-const queryAllByPlaceholderText = queryAllByAttribute.bind(null, 'placeholder')
-
+function queryAllByPlaceholderText(...args) {
+  return queryAllByAttribute('placeholder', ...args)
+}
```

Comment:
ICYW: all of the other queries use named functions and the suggestion feature needs them to be named.